contracts/adapters/ProtocolAdapter.sol

```diff
@@ -174,6 +174,24 @@ library ProtocolAdapter {
                 )
             );
         require(success, getRevertMsg(result));
+        return abi.decode(result, (uint256));
+    }
+
+    function delegateRewardsClaimable(
```

Comment:
We can remove this function completely, because it doesn't need to query any of the variables on `address(this)`.

The RibbonVol contract can just call `adapter.rewardsClaimable(rewardsAddress, optionIDs)` directly and it should work the same. Removes the extra overhead on gas.

Reply:
refer below